client/src/components/App.js

```diff
@@ -7,7 +7,8 @@ import "../assets/scss/main.scss";
 import RegistrationForm from "./registration/RegistrationForm";
 import SignInForm from "./authentication/SignInForm";
 import TopBar from "./layout/TopBar";
-import HolidayList from './HolidayList.js'
+import HolidayList from './HolidayList'
```

Comment:
We want to be consistent in our import statements (and code altogether)-
* Decide between single and double quotations
* Decide between semi-colon or no semi-colon at the end of lines

Once you make a decision, these should be consistent throughout the codebase